src/components/Introduction/StartCreate.jsx

```diff
@@ -31,6 +31,9 @@ const StartCreate = () => (
         <button type="button" className="button">Suivant</button>
       </Link>
       <p className="pWconnect">continuer sans se connecter</p>
+      <Link to="/registerForm">
```

Comment:
`/register`

Reply:
I changed the "route path" to register because there is already a "route" for
register

Reply:
"registerStep"